### PR TITLE
Update to 1.19.4

### PR DESCRIPTION
**Pull request by triphora**

### CODE REVIEW

**Comment by Fabian42** [comment]
I'm getting "`@Inject annotation on getKey could not find any targets matching 'Lnet/minecraft/class_437;keyPressed(III)Z' in net.minecraft.class_437.`". But maybe I did something wrong? The file name still contains "1.19.3". I used `gradle build`.

[latest.log](https://github.com/Minenash/RebindAllTheKeys/files/11117085/latest.log)

File: gradle.properties [hunk 1]
```diff
@@ -3,15 +3,14 @@ org.gradle.jvmargs=-Xmx1G
...
 	# check these on https://fabricmc.net/develop
-	minecraft_version=1.19.3
-	yarn_mappings=1.19.3+build.3
-	loader_version=0.14.11
+	minecraft_version=1.19.4
+	yarn_mappings=1.19.4+build.1
+	loader_version=0.14.18
 
 # Mod Properties
 	mod_version = 1.2.1+1.19.3
```
Comment (triphora) [inline gradle.properties]:
```suggestion
	mod_version = 1.2.1+1.19.4
```

**Comment by Fabian42** [comment]
Yes, I did do something wrong… I downloaded the old state. 🤦 I didn't notice that this wasn't merged yet.
Now it works. Thanks for the update!

File: src/main/java/com/minenash/rebind_all_the_keys/mixin/ClientPlayerEntityMixin.java [hunk 2]
```diff
@@ -14,11 +14,9 @@ public abstract class ClientPlayerEntityMixin {
 
     @Shadow @Final protected MinecraftClient client;
 
-    @Shadow public abstract void setSprinting(boolean sprinting);
-
     @Redirect(method = "tickMovement", at = @At(value = "INVOKE", ordinal = 0, target = "Lnet/minecraft/client/network/ClientPlayerEntity;setSprinting(Z)V"))
     private void disableDoubleTapSprint(ClientPlayerEntity instance, boolean sprinting) {
         if (RebindAllTheKeys.doubleTapSprint.getValue() || client.options.sprintKey.isPressed())
-            setSprinting(true);
+            ((ClientPlayerEntity)(Object)this).setSprinting(true);
```
Comment (Minenash) [inline src/main/java/com/minenash/rebind_all_the_keys/mixin/ClientPlayerEntityMixin.java:20]: Is the a reason to do one over the other?

Comment (triphora) [reply src/main/java/com/minenash/rebind_all_the_keys/mixin/ClientPlayerEntityMixin.java:20]: I do not remember, I was probably getting errors with the existing shadow method and decided to remove it

**Pull request merged**